network-api/networkapi/mozfest/models.py

```diff
@@ -63,12 +63,25 @@ def get_template(self, request):
 
         return 'mozfest/mozfest_primary_page.html'
 
-    def get_context(self, request):
+    def get_context(self, request, bypass_menu_buildstep=False):
         context = super().get_context(request)
         context = get_page_tree_information(self, context)
+
         # Also make sure that these pages always tap into the mozfest newsletter for the footer!
         mozfest_footer = Signup.objects.filter(name__iexact='mozfest').first()
         context['mozfest_footer'] = mozfest_footer
+
+        # Find the homepage, and then record all pages that should end up as nav items. Note
+        # that subclasses can bypass this, because the MozfestHomepage doesn't need any of
+        # this work to be done.
+        if not bypass_menu_buildstep:
+            homepage = list(filter(
```

Comment:
I really wish python has a nicer syntax for filtering out data, but I _think_ this is the best we can do without using a for loop...

Reply:
TIL about `lambda` functions in Python.